index.php

```diff
@@ -0,0 +1,30 @@
+<?php require_once 'trainingFunction.php';
+$all_workouts_array = dbPull();
+$date_array = getAllDates($all_workouts_array);
+$all_workouts_array = addHtmlToWorkouts($all_workouts_array);
+
+?>
+<html lang = "en">
+    <head>
+        <link rel="stylesheet" href="style.css" type="text/css">
+        <link rel="stylesheet" type="text/css" href="normalize.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Mouse+Memoirs&family=Roboto:wght@300;700&display=swap" rel="stylesheet">
+        <title>Training Log</title>
+    </head>
+    <body>
+        <header>
+            <nav class="nav_bar">
+                <a href="#" class="nav_button">Add a session</a>
+            </nav>
```

Comment:
it would be better to remove this and add it in story 2 because at the moment it doesn't do anything and we only want finished features on the `main` branch